test/modules/engin/pkg_module_res.chpl

```diff
@@ -1,5 +1,5 @@
-use AutoMath; //doesn't make a difference -- works either way
-writeln(AutoMath.pi);
+use Math;
+writeln(Math.pi);
```

Comment:
@e-kayrakli - I was curious if this seemed like the right update for this test you wrote.  I could see these lines being intended as using a module that was already included by default, in which case I should find a different symbol to fulfill that role?

Also, would you be willing to review the PR as a whole?

Reply:
Phew, a test from 2016.. The motivating issue for this seemed to be about function resolution with package modules that have been fixed (this was first merged as a future). Probably it shouldn't have been changed to AutoMath in the first place. If anything, this is the way I implemented it first, it seems.

Reply:
True, but Math was included by default then (since it pre-dated the addition of AutoMath)